### DISCUSSION TITLE
LeafLet

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#19341] [Q&A]
Is it possible to add leaftlet maps to osmand as online maps?

just example
https://developers.arcgis.com/esri-leaflet/samples/clustering-feature-layers/

**Comment by ThomasMeister123** [comment]
Waiting for the "POI-Grouping" like this for years. The only reason why I still have LocusMap on my phone

**Comment by vshcherb** [comment]
That means it would mean displaying Web browser instead of maps, doesn't seem possible for now and means most of benefits of Rich client will be lost comparing to Thin client

**Reply by mariush444** [reply]
Does it mean that a system webView should be used?

**Reply by vshcherb** [reply]
Yes, I don't see any other solution. So I don't think this is applicable to OsmAnd
